romemul/config.c

```diff
@@ -24,7 +24,8 @@ static ConfigEntry defaultEntries[MAX_ENTRIES] = {
     {"WIFI_SCAN_SECONDS", TYPE_INT, "15"},
     {"WIFI_PASSWORD", TYPE_STRING, ""},
     {"WIFI_SSID", TYPE_STRING, ""},
-    {"WIFI_AUTH", TYPE_INT, ""}};
+    {"WIFI_AUTH", TYPE_INT, ""},
+    {"WIFI_COUNTRY", TYPE_STRING, ""}};
```

Comment:
We should also consider the empty string as an alias of the `XX` default value worldwide country.

Reply:
empty configuration or wrong value will fallback to XX